src/routes.js

```diff
@@ -94,9 +95,14 @@ export default [
     path: '/admin',
     component: Admin,
   },
+  {
+    name: 'Timesheets',
+    path: '/t1m35433t5',
```

Comment:
The path attribute will be changed when timesheets goes fully public, correct?

Reply:
Yes, we will change it to /timesheets instead of the one that is there now. We made it much less guessable while we are in the testing phase